2022 Structured Data chapter (#3094)
HTTPArchive/almanac.httparchive.org: src/content/en/2022/structured-data.md

  sheets_gid="1452747230",
  sql_file=""
  )
}}

{{ figure_markup(
  image="open-graph-usage-by-year-desktop.png",
  caption="Open Graph usage by year - Desktop",
  description="A year on year comparison of Open Graph usage on desktop pages",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRr8oRb8NNs5MbpPpDi7QSsTUTPQL_vxQtQvmn52qGk97gpNhJIHa8VF0x5ZJtWXAFuEbDFZQB6TIuN/pubchart?oid=1865365764&format=interactive",
  sheets_gid="1452747230",
  sql_file=""
  )
}}

This is matched by desktop, going from 1.57% to 5.38% over the course of the year. We have seen an increase in use for each Open Graph type in the top 10 for both mobile and desktop, resulting in Open Graph’s relative growth of 1.46% since 2021.

### Twitter

{{ figure_markup(
  image="twitter-meta-tag-usage-by-year.png",
  caption="Twitter meta tag usage by year",
  description="A year on year comparison of twitter meta tag usage",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRr8oRb8NNs5MbpPpDi7QSsTUTPQL_vxQtQvmn52qGk97gpNhJIHa8VF0x5ZJtWXAFuEbDFZQB6TIuN/pubchart?oid=270505823&format=interactive",
  sheets_gid="1452747230",
  sql_file=""
  )
}}

> Twitter meta tags once again follow the pattern of a general increase in usage, more specifically in the common tags of `twitter:card`, `twitter:title`, ` twitter:description` and `twitter:image`. A notable increase can be seen for `twitter:label1` and `twitter:data1`, both at 6.85% in 2021 to 9.71% in 2022 for mobile pages.

Author: ```suggestion
Twitter meta tags once again follow the pattern of a general increase in usage, more specifically in the common tags of `twitter:card`, `twitter:title`, `twitter:description` and `twitter:image`. A notable increase can be seen for `twitter:label1` and `twitter:data1`, both at 6.85% in 2021 to 9.71% in 2022 for mobile pages.
```

  sheets_gid="1452747230",
  sql_file=""
  )
}}

Most of the properties for Microdata have not seen much change, with a slight increase in some of the more common properties such as `webpage`, `sitenavigationelement` and `wpheader` appearing in 7.85%, 6.11% and 5.32% of mobile pages respectively.

{{ figure_markup(
  image="microdata-usage-by-year-mobile.png",
  caption="Microdata usage by year - Mobile",
  description="A year on year comparison of microdata usage on mobile pages",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRr8oRb8NNs5MbpPpDi7QSsTUTPQL_vxQtQvmn52qGk97gpNhJIHa8VF0x5ZJtWXAFuEbDFZQB6TIuN/pubchart?oid=1393933149&format=interactive",
  sheets_gid="1452747230",
  sql_file=""
  )
}}

{{ figure_markup(
  image="microdata-usage-by-year-desktop.png",
  caption="Microdata usage by year - Desktop",
  description="A year on year comparison of microdata usage on desktop pages",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRr8oRb8NNs5MbpPpDi7QSsTUTPQL_vxQtQvmn52qGk97gpNhJIHa8VF0x5ZJtWXAFuEbDFZQB6TIuN/pubchart?oid=567931548&format=interactive",
  sheets_gid="1452747230",
  sql_file=""
  )
}}

>  These increases are common for desktop as well, with slight decreases elsewhere such as `wpsidebar` (going from 1.38% to 1.17% on mobile pages and going from 1.25% to 1.08% on desktop pages), resulting in minimum change over the last year for both mobile and desktop pages as a whole.

Author: ```suggestion
These increases are common for desktop as well, with slight decreases elsewhere such as `wpsidebar` (going from 1.38% to 1.17% on mobile pages and going from 1.25% to 1.08% on desktop pages), resulting in minimum change over the last year for both mobile and desktop pages as a whole.
```

With data linking, we collect information from different sources to create richer and more useful data. This is possible thanks to structured data, whose global, unique identifiers allow machines to read and understand the relationship between different types of data. This has the use of creating a more connected web of relationships.

#### Search Engine Optimization & Discoverability

Search engine optimization (<a hreflang="en" href="https://www.webopedia.com/definitions/seo/#How_does_SEO_work">SEO</a>) is the area focusing on building the content of a web page so that it has better results from search engines. Naturally, this is highly important for discoverability as a successful implementation of SEO may allow for a page to appear higher on the search engine results page (<a hreflang="en" href="https://www.webopedia.com/definitions/serp/">SERP</a>). The SERP is where the titles, URLs, and meta descriptions are displayed from a search query.

> By adding structured data to web pages, we can optimize a web page for search engines, as well as have extra content visible from the SERP. This extra content can come in many forms, some of which has been [discussed previously](#heading=h.pun7uv6k8bah), namely Knowledge Panels, Rich Snippets and Related Questions.

Author: ```suggestion
By adding structured data to web pages, we can optimize a web page for search engines, as well as have extra content visible from the SERP. This extra content can come in many forms, some of which has been [discussed previously](#semantic-search-engines-rich-results-and-beyond), namely Knowledge Panels, Rich Snippets and Related Questions.
```
